tests/smoke_tests/parcel/yang_et_al_2018/test_just_do_it.py

```diff
@@ -15,7 +15,7 @@
 def test_just_do_it(scheme, adaptive, backend_class=CPU):
```

Comment:
what is this test about?

Reply:
just executes the simulation and does some basic asserts, the aim is to try it with both SciPy and PySDM condenstion solvers (and in future, also with the GPU backend), which is not covered by the example notebook execution